buildtest/buildsystem/base.py

```diff
@@ -69,28 +69,28 @@ def __init__(self, config_file):
             f"buildtest found the available schema: {self.lookup} in schema library"
         )
 
-        # Load the configuration file, fails on any error
-        self.load(config_file)
+        # Load the Buildspec file, fails on any error
+        self.load(buildspec_file)
 
-    def load(self, config_file):
-        """Load a config file. We check that it exists, and that it is valid.
+    def load(self, buildspec_file):
+        """Load a Buildspec file. We check that it exists, and that it is valid.
            we exit on any error, as the config is not loadable.
 
            Parameters:
 
-           config_file: the pull path to the configuration file, must exist.
+           buildspec_file: the pull path to the configuration file, must exist.
         """
-        self.config_file = os.path.abspath(config_file)
+        self.buildspec_file = os.path.abspath(buildspec_file)
 
-        if not os.path.exists(self.config_file):
-            sys.exit("Build configuration %s does not exist." % self.config_file)
+        if not os.path.exists(self.buildspec_file):
+            sys.exit("BuildSpec File: %s does not exist." % self.buildspec_file)
 
-        elif os.path.isdir(self.config_file):
+        elif os.path.isdir(self.buildspec_file):
             sys.exit(
```

Comment:
For a lot of these saying “Buildspec file” is redundant, a buildspec is a file.

Reply:
I will fix this and change to ``self.buildspec``